[Keyboard] Add Support for PH-AC Rhythm Game Controller (#24368)
qmk/qmk_firmware: keyboards/phdesign/phac/readme.md

# phac

> https://imgur.com/a/iWBXsC0

Reviewer: ```suggestion
![phac](https://i.imgur.com/bQdYGMf.png)
```

> *A short description of the keyboard/project*

Reviewer: This should be filled out.

* Keyboard Maintainer: [Xufeng Tao](https://github.com/nonameCCC)
* Hardware Supported: RP2040 minimal system

> * Hardware Availability: https://m.tb.cn/h.gLFXLaX?tk=7DpL3TuPVxx(currently not available outside China Mainland, as we could not afford the international express fee)

Reviewer 2: Without the space after the link, the `(currently` is getting appended to the URL. 
Also, "currently no available outside of China" (or similar) is sufficient. Additional reasons don't need to be mentioned (part of the reason for the links is for hardware verification and potential acquisition for users)
```suggestion
* Hardware Availability: https://m.tb.cn/h.gLFXLaX?tk=7DpL3TuPVxx (currently not available outside of mainland China)
```

> Make example for this keyboard (using QMK MSYS):
> 
>     qmk compile -kb phdesign/phac -km via
> 
> Flashing example for this keyboard:
> 
>     Just enter the bootloader and put the .uf2 file into the USB storage device that appeared
> 
> See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
> 
> ## Bootloader
> 
> Enter the bootloader in 3 ways:
> 
> * **Bootmagic reset**: Hold down the key at (0,5) in the matrix (the START button) and plug in the controller
> * **Physical reset button**: Short pin hole SW1 on the back of the PCB (not recommended as the case is not easy to open)

Reviewer: ```suggestion
Make example for this keyboard (after setting up your build environment):

    make phdesign/phac:default

Flashing example for this keyboard:

    make phdesign/phac:default:flash

See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).

## Bootloader

Enter the bootloader in 2 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (the START button) and plug in the controller
* **Physical reset button**: Short pin hole SW1 on the back of the PCB (not recommended as the case is not easy to open)
```
